core/sql/sqlcomp/CmpSeabaseDDLmd.h

```diff
@@ -661,6 +690,452 @@ static const MDTableInfo allMDHistInfo[] = {
+  {"   colcount int not null not serialized, "},
+  {"   interval_count smallint not null not serialized, "},
+  {"   rowcount largeint not null not serialized, "},
+  {"   total_uec largeint not null not serialized, "},
+  {"   stats_time timestamp(0) not null not serialized, "},
+  {"   low_value varchar(250) character set ucs2 not null not serialized, "},
+  {"   high_value varchar(250) character set ucs2 not null not serialized, "},
+  {"   read_time timestamp(0) not null not serialized, "},
+  {"   read_count smallint not null not serialized, "},
+  {"   sample_secs largeint not null not serialized, "},
+  {"   col_secs largeint not null not serialized, "},
+  {"   sample_percent smallint not null not serialized, "},
+  {"   cv double precision not null not serialized, "},
+  {"   reason char(1) character set iso88591 not null not serialized, "},
+  {"   v1 largeint not null not serialized, "},
+  {"   v2 largeint not null not serialized, "},
+  {"   v3 largeint not null not serialized, "},
+  {"   v4 largeint not null not serialized, "},
+  {"   v5 varchar(250) character set ucs2 not null not serialized, "},
+  {"   v6 varchar(250) character set ucs2 not null not serialized "},
+  {" , constraint "HBASE_HIST_PK" primary key "},
+  {"     (table_uid, histogram_id, col_position) "},
+  {" ) "},
+  {" ; "}
+};
+
+static const QString seabaseOldTrafMDv11HistogramIntervalsDDL[] =
+{
+  {" create table "HBASE_HISTINT_NAME"_OLD_MD "},
+  {" ( "},
+  {"   table_uid largeint not null not serialized, "},
+  {"   histogram_id int unsigned not null not serialized, "},
+  {"   interval_number smallint not null not serialized, "},
+  {"   interval_rowcount largeint not null not serialized, "},
+  {"   interval_uec largeint not null not serialized, "},
+  {"   interval_boundary varchar(250) character set ucs2 not null not serialized, "},
+  {"   std_dev_of_freq numeric(12,3) not null not serialized, "},
+  {"   v1 largeint not null not serialized, "},
+  {"   v2 largeint not null not serialized, "},
+  {"   v3 largeint not null not serialized, "},
+  {"   v4 largeint not null not serialized, "},
+  {"   v5 varchar(250) character set ucs2 not null not serialized, "},
+  {"   v6 varchar(250) character set ucs2 not null not serialized"},
+  {" , constraint "HBASE_HISTINT_PK" primary key "},
+  {"    (table_uid, histogram_id, interval_number) "},
+  {" ) "},
+  {" ; "}
+};
+
+////////////////////////////////////////////////////////////////////////
+//// END_OLD_MD_v11: 
+////        OLD metadata Version 1.1  (Major version = 1, Minor version = 1).
+//////////////////////////////////////////////////////////////////////////
+
+// Note: The older metadata definitions below were from before we
+// reset the version to 1.1 with the early Trafodion releases. (We
+// went backwards to match the Trafodion release number.)
+
+////////////////////////////////////////////////////////////////////////
 //// START_OLD_MD_v23: 
```

Comment:
Is there any plan to remove old versions or do we plan to just keep tacking on new versions?